util/fpga/cw_spiflash.py

```diff
@@ -179,12 +189,15 @@ def __init__(self, fpga, board='CW305'):
         self._fpga_io = fpga.gpio_mode()
         self._fpga_io.pin_set_output(self._PIN_MAPPINGS[self._board].PIN_TRST)
         self._fpga_io.pin_set_output(self._PIN_MAPPINGS[self._board].PIN_SRST)
-        self._fpga_io.pin_set_output(self._PIN_MAPPINGS[self._board].PIN_JTAG_SPI)
-        self._fpga_io.pin_set_output(self._PIN_MAPPINGS[self._board].PIN_BOOTSTRAP)
+        self._fpga_io.pin_set_output(self._PIN_MAPPINGS[self._board].PIN_TAP_STRAP1)
+        self._fpga_io.pin_set_output(self._PIN_MAPPINGS[self._board].PIN_SW_STRAP0)
```

Comment:
do we need to set the other strap pins to output also?

Reply:
Oh yes good catch. Let me do that.